### ISSUE TITLE
Regression introduced when upgrading from 1.12.1 to 1.13.1

### ISSUE THREAD

**Issue by pun-ky** [jhy/jsoup#1350]
Only version upgraded then:

```kotlin
Caused by: java.lang.NullPointerException
        at org.jsoup.nodes.Element.isInlineable(Element.java:1619)
        at org.jsoup.nodes.Element.outerHtmlHead(Element.java:1479)
        at org.jsoup.nodes.Node$OuterHtmlVisitor.head(Node.java:740)
        at org.jsoup.select.NodeTraversor.traverse(NodeTraversor.java:25)
        at org.jsoup.nodes.Node.outerHtml(Node.java:606)
        at org.jsoup.nodes.Node.outerHtml(Node.java:601)
        at org.jsoup.nodes.Node.toString(Node.java:635)
        at com.cognifide.gradle.aem.common.pkg.vault.FilterElement.toString(FilterElement.kt:37)
        at com.cognifide.gradle.aem.common.pkg.vault.VaultDefinition.getFilters(VaultDefinition.kt:80)
        at com.cognifide.gradle.aem.common.pkg.vault.VaultDefinition$Del
```

Related code / I am using Jsoup as markup builder: https://github.com/Cognifide/gradle-aem-plugin/blob/master/src/main/kotlin/com/cognifide/gradle/aem/common/pkg/vault/FilterElement.kt#L19-L37

**Comment by pun-ky** [comment]
btw how to render `Element` as self-closed tag? :)

**Comment by jhy** [comment]
Fixed / dupe of #1410
